cmd/root.go

```diff
@@ -178,53 +179,50 @@ func initZCNCoreContext() {
 func initZwalletContext() {
 	// create wallet
 	if !walletIsLoaded {
-		createWallet()
-		loadWallet()
+		createNLoadWallet()
 		walletIsLoaded = true
 	}
 }
 
-func createWallet() {
-
-	_, err := os.Stat(cfgWallet)
-
-	isNewWallet := os.IsNotExist(err)
-
-	if isNewWallet {
+func createNLoadWallet() {
```

Comment:
Let's use createAndLoadWallet() to make it more readable.

Reply:
Sure. Thanks